src/types/dm_channel.rs

```diff
@@ -0,0 +1,17 @@
+#[derive(Deserialize, Clone)]
+pub struct DMChannel {
+    /// The channel ID
+    ///
+    /// 128 bit unsigned integer
+    pub id: u128,
+
+    /// The channel's name
+    /// Only applies to Group DMs
+    /// String of max length 100 chars
+    pub name: Option<String>,
+
+    /// The users inside the channel
+    ///
+    /// Vec of 128 bit unsigned integers
+    pub users: Vec<u128>,
```

Comment:
Either make this `Vec<User>` or manually implement Serialize. I'm not sure how to implement Serialize for a Vec so you would be on your own if you chose to go that route.

Reply:
It would probably also be worthwhile to send the user objects here anyways.

Reply:
alright I'll switch to Vec<User>